zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -1226,12 +1226,6 @@ protected Election createElectionAlgorithm(int electionAlgorithm) {
 
         //TODO: use a factory rather than a switch
         switch (electionAlgorithm) {
```

Comment:
- Now we can remove this `TODO`
- Now we don't need this `switch-case ` structure, only support `Algorithm3`

Reply:
Hey @maoling I have made addressed your comments and please do let me know if you see any additional changes that are required.